cardano-cli/src/Cardano/CLI/EraBased/Run/Transaction.hs

```diff
@@ -167,13 +167,20 @@ runTransactionBuildCmd
 
   -- Conway related
   votingProcedures <-
-    inEonForEra
-      (pure emptyVotingProcedures)
-      (\w -> firstExceptT TxCmdVoteError $ ExceptT (readVotingProceduresFiles w voteFiles))
-      eon
+    case mfVoteFiles of
```

Comment:
So we're now pattern matching twice essentially based on whether or not a feature is in an era. Return the empty list instead of `!(Maybe (Featured ConwayEraOnwards era [VoteFile In]))`

Reply:
Can you clarify what you mean?  Is the problem the case match n the `Maybe`, once for voting procedures and once for proposals?